src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommand.cs

```diff
@@ -358,10 +358,10 @@ async Task<IEnumerable<string>> GetPackageDownloadUrlsAsync(IEnumerable<Workload
             }
         }
 
-        private async Task UseTempManifestsToResolvePacksAsync(DirectoryPath tempPath, bool includePreview)
+        async Task UseTempManifestsToResolvePacksAsync(DirectoryPath tempPath, bool includePreview)
```

Comment:
This goes against coding style <https://github.com/dotnet/runtime/blob/a55d5300457004aff05a603ba9960290517f5681/docs/coding-guidelines/coding-style.md?plain=1#L14-L16>